src/jimgw/cli/_likelihood.py

```
@@ -117,6 +123,68 @@ def build_likelihood(
+                    p
+                    for p in prior.base_prior
+                    if "t_det" in p.parameter_names and hasattr(p, "xmin")
+                ),
+                None,
+            )
+            if tdet_comp is not None:
+                data = ifos[0].data
+                t_end = float(data.start_time) + float(data.duration) - trigger_time
+                s = EARTH_RADIUS_LIGHT_S
+                if mb_time_offset is None:
+                    mb_time_offset = t_end - float(getattr(tdet_comp, "xmin")) + s
```

Ruff (0.15.12)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)